README.md

```diff
@@ -1,2 +1,30 @@
-# awesome-books-with-es6
- Awesome Books project with ES6
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome Books
+
+ "Awesome books" is a simple website that displays a list of books and allows you to add and remove books from that list. By building this application, you will learn how to manage data using JavaScript. Thanks to that your website will be more interactive. You will also use a medium-fidelity wireframe to build the UI.
+
+## Built With
+
+- Major languages - HTML, CSS & JS
+
+## Live Demo (if available)
+
+[Live Demo Link](https://walterokumu.github.io/awesome-books/)
+
+## Authors
+
+👤 **Walter Okumu**
+
+- GitHub: [@WalterOkumu](https://github.com/WalterOkumu)
+- Twitter: [@OkumuOriaro](https://twitter.com/OkumuOriaro)
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
```

Comment:
- [ ] I suggest you add in your README file the section with instructions about how to set up your project on the local machine 👍